Lesson27/ezgi/create/create-post.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Create post</title>
+		<link rel="stylesheet" type="text/css" href="../styles.css" />
+	</head>
+	<body>
+		<div class="container">
+			<h1>Create post</h1>
+      
+		<form action="" id="create-post">
+			<label for="title">Title :</label>
+			<textarea id="title"></textarea>
+
+			<label for="body">Paragraph</label>
+			<textarea id="body"></textarea>
+			<button id="Create-post-button" type="submit">Create Post</button>
```

Comment:
It would be better to rename this id to be in lowercase but make sure to update all the usages of it.
```
			<button id="create-post-button" type="submit">Create Post</button>
```